tox.ini

```diff
@@ -182,6 +182,7 @@ extras =
     doc
 commands =
     bash -c 'rm -rf docs/_build docs/api/*.rst'
+    datapkg_to_rst
```

Comment:
Can do -- though I also wrote the metadata script to overwrite any current script.

Reply:
I don't fully understand the reasoning behind specifying the location of the datapackage.json file in the argument. Is this an alternative to specifying the path as a global variable in the module itself?

Reply:
do you think this is better than using the `import importlib.resources`?

Reply:
So right now, this is a script that can do exactly one thing -- it can transform a single file, in a single hard-coded location, into another single file in another single hard-coded location. If either of those locations changes, it'll stop working, and the code itself will need to be edited. In general it's better to have the code specify a set of operations which can be applied to a variety of inputs, generating a variety of outputs. In this case the input would be any datapackage descriptor, and the output would be an RST file, with both of them being specified as command line arguments. This would also make it clearer to someone reading the `tox.ini` file what the script is doing -- which files it's operating on -- without their needing to open up the code to see what it's doing. So it would look like:

```
datapkg_to_rst src/pudl/package_data/meta/datapkg/datapackage.json > docs/data_sources/metadata.rst
```
The name of the script says what it does pretty clearly, and you can see which datapackage descriptor it's operating on, and what RST file it is creating (the `>` character says "redirect output from the preceding command to the following file"

This behavior is in line with how scripts typically behave. We know that this script is being called by `tox` and we know that `tox` is going to be run from within the repository, as opposed to being run by a user who has installed the package via pip/conda, so it's safe in this context to refer to the `package_data` path and the `docs` path, whereas previously the code was referring to these paths internally -- a context in which you couldn't be sure whether the user would have access to the whole repo, or whether it would just be the installed package. I think this is preferable to using the `importlib.resources` access mode.

Reply:
fixed so that it takes ` datapkg_to_rst -i src/pudl/package_data/meta/datapkg/datapackage.json -o docs/data_sources/metadata.rst
`